Tests/Runtime/MobiledgeX_RuntimeTests.cs

```diff
@@ -60,9 +60,9 @@ public void Clean()
         [TestCase("MobiledgeX", "MobiledgeX SDK Demo", "2.0")]
         public void RegisterClient(string orgName, string appName, string appVers)
         {
-            MobiledgeXIntegration.orgName = orgName;
-            MobiledgeXIntegration.appName = appName;
-            MobiledgeXIntegration.appVers = appVers;
+            integration.orgName = orgName;
+            integration.appName = appName;
+            integration.appVers = appVers;
```

Comment:
Because you switched away from static variables, I think you can just create a MobiledgeXIntegration instance in each of your test cases instead of changing the vars each time.

Reply:
Will redo in a separate PR.

Reply:
++

For test cases, separate independent instances just makes the tests more self contained.